migrations/4_genesis.js

```diff
@@ -67,22 +95,16 @@ module.exports = function(deployer, network, accounts) {
             return genesisManager.addCommunityGrant(grant.receiver, grant.amount)
         }))
 
-        deployer.logger.log("Finalizing dummy token distribution...")
-
-        await dummyTokenDistribution.finalize()
-
         deployer.logger.log("Ending genesis and transferring ownership of the LivepeerToken to the protocol Minter...")
 
         await genesisManager.end()
 
-        if (!lpDeployer.isProduction(network)) {
-            // If not in production, the deployment account holds the company supply
-            // and use some of those tokens to fund the faucet supply
-            deployer.logger.log("Not in production - deployment account will fund the deployed faucet")
+        if (lpDeployer.isProduction(network)) {
+            // If in production, start generation period
+            deployer.logger.log("In production - starting generation period")
+            const merkleMine = await MerkleMine.deployed()
+            await merkleMine.start()
```

Comment:
The MerkleMine contract would've already received the necessary tokens after `genesisManager.end()` so we can call `start()` here